content/millennium/r4/financial/account.md

```diff
@@ -19,8 +19,10 @@ The Account resource acts as a central record against which charges, payments, a
 
 The account resource supports multiple account types which are defined below.
 
+* `charge-group` - Charge group.
```

Comment:
We are still in discussion about what to write about the `charge-group`.

Reply:
Thanks for making the change. I was thinking the same thing as far as them not being full sentences.  I'm fine either way as long as they are  all there or all gone.

Reply:
Updated - https://github.com/cerner/fhir.cerner.com/pull/664/commits/2f7802952ccfdee91802cf56a58579f52868e24f#diff-c8eb47813f9124e188b80dbfe03423839685e5b0f517fa4bcf1a8222a4db0570R22

Reply:
I heard back from CPA and here is what they came up with "A grouping of charges for billing"

Reply:
Updated - https://github.com/cerner/fhir.cerner.com/pull/664/commits/d4ed7ba6b41eae54afb2ac61606e5c9098c664a5#diff-c8eb47813f9124e188b80dbfe03423839685e5b0f517fa4bcf1a8222a4db0570R22

Reply:
+1